src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -93,7 +93,7 @@ internal partial class Binder
             }
 
             if (conversion.IsTupleLiteralConversion ||
-                (conversion.Kind == ConversionKind.ImplicitNullable && conversion.UnderlyingConversions[0].IsTupleLiteralConversion))
+                (conversion.IsNullable && conversion.UnderlyingConversions[0].IsTupleLiteralConversion))
```

Comment:
>(conversion.IsNullable && conversion.UnderlyingConversions[0].IsTupleLiteralConversion)) [](start = 16, length = 88)

Do we have tests for nullable conversions that involve user-defined conversions?